docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -435,7 +435,10 @@ resource "mongodbatlas_advanced_cluster" "test" {
 
 While the example initially defines 2 symmetric shards, auto-scaling of `electable_specs` or `analytic_specs` can lead to asymmetric shards due to changes in `instance_size`.
 
--> **NOTE:** After you upgrade to version 1.23.0 of the provider, you must update the cluster configuration to activate the auto-scaling per shard feature.
+-> **NOTE:** There are specific scenarios where a `mongodbatlas_advanced_cluster` using the new sharding configuration (single replication_specs per shard) might not have shard-level auto-scaling enabled:
+1. Configuration was defined prior to version 1.23.0 when auto-scaling per shard feature was released
+2. Cluster was imported from a legacy schema (e.g., mongodbatlas_cluster or mongodbatlas_advanced_cluster using num_shards > 1)
+In these cases, you must update the cluster configuration to activate the auto-scaling per shard feature. This can be done by temporarily modifying a value like compute_min_instance_size.
```

Comment:
```suggestion
1. Configuration was defined prior to version 1.23.0 when auto-scaling per shard feature was released.
2. Cluster was imported from a legacy schema (For example, `mongodbatlas_cluster` or `mongodbatlas_advanced_cluster` using `num_shards` > 1).
In these cases, you must update the cluster configuration to activate the auto-scaling per shard feature. This can be done by temporarily modifying a value like `compute_min_instance_size`.
```